ci/docker/python-free-threaded-wheel-musllinux-test-imports.dockerfile

```
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+ARG base
+FROM ${base}
+
+RUN apk add --no-cache bash build-base bzip2-dev dpkg dpkg-dev g++ git libffi-dev libnsl-dev libtirpc-dev \
+    linux-headers ncurses-dev openssl-dev pkgconf tzdata zlib-dev
```

Could you use one line per package for easy to read?

```suggestion
RUN apk add --no-cache \
      bash \
      build-base \
      bzip2-dev \
      dpkg \
      dpkg-dev \
      g++ \
      git \
      libffi-dev \
      libnsl-dev \
      libtirpc-dev \
      linux-headers \
      ncurses-dev \
      openssl-dev \
      pkgconf \
      tzdata \
      zlib-dev
```